skbio/stats/composition.py

```diff
@@ -896,9 +948,27 @@ def ancom(table, grouping,
         else:
             nu = cutoff[4]
         reject = (W >= nu*n_feat)
-    labs = mat.columns
-    return pd.DataFrame({'W': pd.Series(W, index=labs),
-                         'reject': pd.Series(reject, index=labs)})
+
+    feat_ids = mat.columns
+    ancom_df = pd.DataFrame({'W': pd.Series(W, index=feat_ids),
+                             'reject': pd.Series(reject, index=feat_ids)})
+
+    if len(percentiles) == 0:
+        return ancom_df, pd.DataFrame()
+    else:
```

Comment:
What do you think about separating this helper function into its own private function?  I could see this functionality becoming useful for other tests (i.e. anosim, permanova).